### PR TITLE
Feat/zapier app

### LINKED ISSUE #1543: Zapier App
send same events as we do with webhooks but natively in Zapier

later: also receive data from zapier (i.e. create an event-type / create an event / etc.)

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?
Adds Zapier App to App Store

**Demo**
https://www.loom.com/share/1f7bc75564014e0fa3e4415b2d416d40

fixes #1543

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Create cal.com Zapier app:
   - Follow these instructions: [https://oasis-aragon-9cb.notion.site/Setup-Zapier-Integration-a086608278754776956a8066ed892666](url)
- Create a new zap with cal.com trigger (use invite link)
- Use generated api key to connect with the cal.com account 
- Finish zap (More details in the demo video)

## Checklist
- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by PeerRich** [comment]
this is super dope! great setup screen (cc @Jaibles – we should use this for other apps too) and great loom! cant wait to have this go live.

do we also plan to have actions? or only triggers for now

**Comment by PeerRich** [comment]
also added you to the calcom organisation so you can now create branches from within the repo vs from the outside fork

**Comment by CarinaWolli** [comment]
todo add i18n

**Comment by CarinaWolli** [comment]
Todo add detailed zapier setup guide to docs

**Comment by CarinaWolli** [comment]
> this is super dope! great setup screen (cc @Jaibles – we should use this for other apps too) and great loom! cant wait to have this go live.
> 
> do we also plan to have actions? or only triggers for now

For right now only triggers, but I'll take a look into actions

**Comment by baileypumfleet** [comment]
The Loom looks great, looking forward to reviewing this properly

**Comment by CarinaWolli** [comment]
> todo add i18n

done

**Comment by CarinaWolli** [comment]
> Todo add detailed zapier setup guide to docs

First version done: [README.md](https://github.com/CarinaWolli/cal.com/blob/feat/zapier-app/packages/app-store/zapierother/README.md)

**Review by alishaz-polymath — DISMISSED** [review]
Code looks good to me. I'm not 100% sure about the `yarn.lock` file though, so I'll just block merge until we get some review from @zomars & @emrysal.
Will test it out in the mean time! Great work 👍

**Comment by CarinaWolli** [comment]
> Code looks good to me. I'm not 100% sure about the `yarn.lock` file though, so I'll just block merge until we get some review from @zomars & @emrysal. Will test it out in the mean time! Great work 👍

I removed the changes from yarn.lock. Shouldn't have committed them, I didn't install any new packages or anything

**Comment by zomars** [comment]
Reviewing now and updating to it gets added to the new DB.

File: apps/web/public/static/locales/en/common.json
Comment (zomars) [inline apps/web/public/static/locales/en/common.json:787]: Should we name space these with `"zapier_`? At least for the steps.

Comment (emrysal) [reply apps/web/public/static/locales/en/common.json:787]: Should the steps be a separate translation for each line? I agree with the above either way

File: packages/prisma/schema.prisma
Comment (zomars) [inline packages/prisma/schema.prisma:395]: Does it makes a difference from normal Webhooks? I'm struggling to understand the need to differentiate other than the payloadTemplate maybe?

File: packages/prisma/seed-app-store.ts
Comment (zomars) [inline packages/prisma/seed-app-store.ts:87]: This is so the app is actually added to the DB

File: packages/prisma/schema.prisma
Comment (zomars) [inline packages/prisma/schema.prisma:420]: BTW we now have an `App` model. It theory we could just add a relationship like: `appId: "zapier"` that way we could have a way to fetch only zapier related api keys without having add a new type.

File: apps/web/public/static/locales/en/common.json
Comment (zomars) [inline apps/web/public/static/locales/en/common.json:781]: Consolidated setup instructions to a single string.

File: apps/web/public/static/locales/es/common.json
Comment (zomars) [inline apps/web/public/static/locales/es/common.json:664]: Also added Spanish translation to make sure it works

File: packages/prisma/schema.prisma
Comment (zomars) [reply packages/prisma/schema.prisma:395]: Gotcha! I have some improvements to make then. Since since the latest release we can just do `prisma.app.findUnique({ where: { slug: "zapier" }})` to check if Zapier is installed now. (We don't check `.env` variables anymore for that)

**Review by zomars — COMMENTED** [review]
Some comments and updates. Overall this is very well done. Just making some adjustments to match our new way to handle app installations.

**Review by zomars — APPROVED** [review]

**Pull request merged**
